src/gui/widgets/FxLine.cpp

```diff
@@ -245,6 +265,40 @@ void FxLine::contextMenuEvent( QContextMenuEvent * )
 
 
 
+void FxLine::changeColor()
+{
+	QColor color = QColorDialog::getColor();
+
+	if ( !color.isValid() )
+	{
+		return;
+	}
+	m_useStyleColor = false;
+
+	FxChannel * targetChannel = Engine::fxMixer()->effectChannel( m_channelIndex );
+
+	targetChannel->m_customColor = color;
+	targetChannel->m_useStyleColor = false;
+
+	m_useStyleColor = false;
+	m_customBackgroundActive.setColor( color );
+	m_customBackground.setColor( color.darker( 150 ) );
+
+	update();
+}
+
+
+
+void FxLine::resetColor()
+{
+	FxChannel * targetChannel = Engine::fxMixer()->effectChannel( m_channelIndex );
+	targetChannel->m_useStyleColor = true;
+	m_useStyleColor = true;
+}
```

Comment:
I think you should add `update();` at the end of this function to fix the bug discovered by @DouglasDGI.